diff --git a/node_modules/grunt-mocha-cli/lib/index.js b/node_modules/grunt-mocha-cli/lib/index.js
index 59f7a07..f01209e 100644
--- a/node_modules/grunt-mocha-cli/lib/index.js
+++ b/node_modules/grunt-mocha-cli/lib/index.js
@@ -2,6 +2,7 @@
 const fs = require('fs')
 const path = require('path')
 const grunt = require('grunt')
+const os = require("os");
 
 const BOOL_OPTIONS = [
   'allow-uncaught',
@@ -70,6 +71,10 @@ module.exports = function(options) {
       })
     }
     
+    if (os.platform() === 'win32') {
+      spawnOptions.opts.shell = true
+    }
+
     if (Array.isArray(options.flags)) {
       args = args.concat(options.flags)
     }
